corehq/apps/locations/tests/test_location_fixtures.py

```diff
@@ -797,57 +797,55 @@ def test_archiving_location_should_resync(self):
 @mock.patch('corehq.apps.domain.models.Domain.uses_locations', lambda: True)
 class LocationFixtureSyncSettingsTest(TestCase):
 
+    @classmethod
+    def setUpClass(cls):
+        super(LocationFixtureSyncSettingsTest, cls).setUpClass()
+        cls.domain = Domain(name=uuid.uuid4().hex)
+        cls.domain.save()
```

Comment:
Nit: I'd prefer `self.domain_obj` for clarity

Reply:
https://github.com/dimagi/commcare-hq/pull/21664/commits/21b044f4aba2c2e29ea76b92f69a2962ea5847cf